### PR TITLE
home ui update blue screen

### PR DESCRIPTION
**Pull request by danezorrilla**
## Ui blue screen bg
- change home ui to background blue
- extract color to colors resources

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -3,4 +3,5 @@
     <color name="colorPrimary">#6200EE</color>
     <color name="colorPrimaryDark">#3700B3</color>
     <color name="colorAccent">#03DAC5</color>
+    <color name="blue">#0000ff</color>
```
Comment (Dalo-Chinkhwangwa-Prof) [inline app/src/main/res/values/colors.xml]: please use upper case for these values - just as the values above. Thanks

**Review by Dalo-Chinkhwangwa-Prof — CHANGES_REQUESTED** [review]
Please review the comments - and apply the changes.
Everything else looks good.

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (danezorrilla) [reply app/src/main/res/values/colors.xml]: I've made the changes you required, please review

Comment (Dalo-Chinkhwangwa-Prof) [reply app/src/main/res/values/colors.xml]: I meant the actual color value

Comment (danezorrilla) [reply app/src/main/res/values/colors.xml]: Changes made

Comment (danezorrilla) [reply app/src/main/res/values/colors.xml]: thanks

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -3,5 +3,5 @@
     <color name="colorPrimary">#6200EE</color>
     <color name="colorPrimaryDark">#3700B3</color>
     <color name="colorAccent">#03DAC5</color>
-    <color name="Blue">#0000ff</color>
+    <color name="Blue">#0000FF</color>
```
Comment (Dalo-Chinkhwangwa-Prof) [inline app/src/main/res/values/colors.xml:6]: looking good (y)

**Review by Dalo-Chinkhwangwa-Prof — APPROVED** [review]
Looks great! Please merge

**Pull request merged**